cephalopod/settings.cfg.example

```diff
@@ -45,3 +45,81 @@ CELERYBEAT_SCHEDULE = {
         'schedule': timedelta(days=1)
     }
 }
+
+# Multipass settings
+
+# User email whitelist
+USER_WHITELIST = {
+    'github': ['gh-username', 'gh-username2'],  # customize access granted GitHub usernames
+    'my-ldap': ['my-ldap-username', 'my-ldap-username2']  # customize access granted my-ldap usernames
+}
+
+# Register your test application here: https://github.com/settings/applications/
+
+_github_oauth_config = {
+    'consumer_key': '',  # put your key here
+    'consumer_secret': '',  # put your secret here
+    'request_token_params': {'scope': 'user:email'},
+    'base_url': 'https://api.github.com',
+    'request_token_url': None,
+    'access_token_method': 'POST',
+    'access_token_url': 'https://github.com/login/oauth/access_token',
+    'authorize_url': 'https://github.com/login/oauth/authorize'
+}
+
+_my_ldap_config = {
+    'uri': 'ldaps://ldap.example.com:636',
+    'bind_dn': 'uid=admin,DC=example,DC=com',
+    'bind_password': 'p455w0rd',
+    'timeout': 30,
+    'verify_cert': True,
+    # optional: if not present, uses certifi's CA bundle (if installed)
+    'cert_file': 'path/to/server/cert',
+    'starttls': False,
+    'page_size': 1000,
+
+    'uid': 'uid',
+    'user_base': 'OU=Users,DC=example,DC=com',
+    'user_filter': '(objectCategory=person)',
+
+    'gid': 'cn',
+    'group_base': 'OU=Organizational Units,DC=example,DC=com',
+    'group_filter': '(objectCategory=groupOfNames)',
+    'member_of_attr': 'memberOf',
+    'ad_group_style': False,
+}
+
+MULTIPASS_AUTH_PROVIDERS = {
+      'github': {
+        'type': 'oauth',
+        'title': 'GitHub',
+        'oauth': _github_oauth_config
+    },
+     'my-ldap': {
+        'type': 'ldap',
+        'title': 'My Organization LDAP',
+        'ldap': _my_ldap_config,
+    },
+}
+
+MULTIPASS_IDENTITY_PROVIDERS = {
+     'github': {
+        'type': 'oauth',
+        'oauth': _github_oauth_config,
+        'endpoint': '/user',
+        'identifier_field': 'login',
+    },
+     'my-ldap': {
+        'type': 'ldap',
+        'ldap': _my_ldap_config,
+    },
+}
+
+MULTIPASS_PROVIDER_MAP = {
+    'github': 'github',
+    'my-ldap': 'my-ldap'
+}
+
+MULTIPASS_LOGIN_SELECTOR_TEMPLATE = 'login_selector.html'
```

Comment:
This and the next one doesn't really belong in the example config - they should be set after loading the config in the code since they aren't supposed to be changed.